schema/causality_jwt.js

```diff
@@ -36,7 +36,9 @@ export default {
       if (options.role === 'OPERATOR' && me.type !== 'Admin') {
         throw new Error('Unauthorized to act as an operator');
       }
-      const registered = find(bidders, (b) => b.sale._id === options.sale_id);
+      const registered = find(bidders, (b) => {
+        return (b.sale._id === options.sale_id || b.sale.id === options.sale_id);
```

Comment:
Totally possible, I just used this because it was a quick fix that worked locally. Resolving the actual ID sounds too complicated for me to do alone, happy to pair. However, the resolution should be done on the metaphysics side, otherwise we loose the benefit of the GraphQL code altogether.